### ISSUE TITLE
mistake double quotation for Help() Command

### ISSUE THREAD

**Issue by sampsonye** [microsoft/power-fx-host-samples#6]
![image](https://user-images.githubusercontent.com/5895634/140877664-34455c28-b9e9-4b84-9b7a-03d31f3ccbe9.png)

I think this is the best case:
```
Record syntax is { < field >: < value >, ... } without quoted field names.
    Example: { Name: "Joe", Age: 29 }
Use the Table function for a list of records.  
    Example: Table( { Name: "Joe" }, { Name: "Sally" } )
Use [ <value>, ... ] for a single column table, field name is "Value".
    Examples: [ 1, 2, 3 ] 
Records and Tables can be arbitrarily nested.
```
![image](https://user-images.githubusercontent.com/5895634/140877927-44105803-fd9c-4860-aa55-907afec4ce44.png)
Why we need format this StringValue?

**Comment by MikeStall** [comment]
Help() should probably just directly print the help string. 
It's returning the help string as a value, which is then going through a formatting in PrintResult(). 

PrintResult() is attempting to print the string as a power fx expression, so it encloses in quotes (and escapes quotes).

**Comment by MikeStall** [comment]
Fixed in #23
